docs/command/mongocli-atlas-metrics.txt

```diff
@@ -49,9 +49,9 @@ Inherited Options
 Related Commands
 ----------------
 
-* :ref:`mongocli-atlas-metrics-databases` - List available databases or databases measurements for a given host.
-* :ref:`mongocli-atlas-metrics-disks` - List available disks or disks measurements for a given host.
-* :ref:`mongocli-atlas-metrics-processes` - Get measurements for a given host.
+* :ref:`mongocli-atlas-metrics-databases` - List available databases or databases metrics for a given host.
```

Comment:
database metrics 

That is, not plural here. List available databases and database metrics for a given host.